srtcore/channel.h

```diff
@@ -114,26 +116,26 @@ class CChannel
       /// Query the socket address that the channel is using.
       /// @param [out] addr pointer to store the returned socket address.
 
-   void getSockAddr(sockaddr* addr) const;
+   void getSockAddr(ref_t<sockaddr_any> addr) const;
 
       /// Query the peer side socket address that the channel is connect to.
       /// @param [out] addr pointer to store the returned socket address.
 
-   void getPeerAddr(sockaddr* addr) const;
+   void getPeerAddr(ref_t<sockaddr_any> addr) const;
```

Comment:
`ref_t`

Reply:
This will be used up to the moment when the initiative of `ref_t` removal is in place. Now please leave it as is. This initiative involves also removal of the `ref_t` definition at all, so rest assured, once it's in place, will embrace that case, too.

Reply:
On the second thought, I think I can apply this defined convention right now as well.